about.html

```diff
@@ -1,73 +1,133 @@
+          collaborative thinking to further the overall value proposition.
+          Organically grow the holistic world view of disruptive innovation via
+          workplace diversity and empowerment.
+        </p>
+      </div>
+      <div class="top-two-boxes">
+        <div class="top-two">
+          <div class="strategy-box">
+            <img class="strategy-img" src="img/about-plan.png" alt="strategy" />
+
+            <h2>Strategy</h2>
+
+            <p>
+              Iterative approaches to corporate strategy foster collaborative
+              thinking to further the overall value proposition.
+            </p>
+
+            <button>Learn More</button>
+          </div>
+        </div>
+
+        <div class="top-two">
+          <div class="how-we-work-box">
+            <img
+              class="working-img"
+              src="img/about-working.png"
+              alt="strategy"
+            />
+
+            <h2>How We Work</h2>
+
+            <p>
+              Iterative approaches to corporate strategy foster collaborative
+              thinking to further the overall value proposition.
+            </p>
+
+            <button>Learn More</button>
+          </div>
+        </div>
+      </div>
+      <div class="bottom-two-boxes">
+        <div class="bottom-two">
+          <div class="places-we-work-box">
+            <img class="office-img" src="img/about-office.png" alt="strategy" />
+
+            <h2>Places We Work</h2>
+
+            <p>
+              Iterative approaches to corporate strategy foster collaborative
+              thinking to further the overall value proposition.
+            </p>
+
+            <button>Learn More</button>
+          </div>
+        </div>
+        <div class="bottom-two">
+          <div class="collaboration-box">
+            <img
+              class="collaboration"
+              src="img/about-meeting.png"
+              alt="strategy"
+            />
+
+            <h2>Collaboration</h2>
+
+            <p>
+              Iterative approaches to corporate strategy foster collaborative
+              thinking to further the overall value proposition.
+            </p>
+
+            <button>Learn More</button>
+          </div>
+        </div>
+      </div>
+      <div class="bottom-content">
+        <h3>Let's Work Together</h3>
+
+        <p class="bottom-text">
+          Leverage agile frameworks to provide a robust synopsis for high level
+          overviews. Iterative approaches to corporate strategy foster
+          collaborative thinking to further the overall value proposition.
+          Organically grow the holistic world view of disruptive innovation via
+          workplace diversity and empowerment.
+        </p>
+      </div>
       <footer>
         <nav>
           <a href="index.html">Home</a>
-          <a href="#">About</a>
+          <a href="file://about.html">About</a>
           <a href="#">Products</a>
```

Comment:
can omit the file:// in the href, i know we talked about it on the index page, just missed fixing it on this one